bindings/hyperdrivepy/src/hyperdrive_state_methods.rs

```diff
@@ -169,6 +170,18 @@ impl HyperdriveState {
         Ok(result)
     }
 
+    pub fn calculate_open_short_deltas(&self, bond_amount: &str) -> PyResult<String> {
+        let bond_amount_fp = FixedPoint::from(U256::from_dec_str(bond_amount).map_err(|_| {
+            PyErr::new::<PyValueError, _>("Failed to convert bond_amount string to U256")
```

Comment:
no, we map error on the U256 from. I guess we could also put a map error on the FixedPoint from, but that's not how we've done it so far.

Reply:
ultimately yeah the thing we're trying to do is convert to fixedpoint and it broke, so I agree it makes sense to have that in the error. But the way we do it everywhere is assume that the error will come from the string->U256 conversion and not the U256->FixedPoint conversion. Since we have the error attached to the string->U256 conversion we say that in the error message.

Not gonna make the change here to keep things consistent, but yeah I think we could do this better in a single PR.

Reply:
https://github.com/delvtech/hyperdrive-rs/issues/65